document/CoreFormat.mdk

```diff
@@ -508,6 +552,43 @@ whose value is a string specifying that path.
 If the location of the file within its containing property cannot be expressed by means of a path,
 then the `pathFromParent` property shall be absent.
 
+If the `file` object occurs as the only member of an array within the `run.files` object (&sect;[#run-files]),
```

Comment:
You can't tell from GitHub's context, but this text is added to the section on `file.pathFromParent`.